va/va.h

```diff
@@ -794,6 +794,19 @@ typedef enum
      *  attribute value is \c VAConfigAttribValContextPriority
      */
     VAConfigAttribContextPriority       = 41,
+
+    /** \brief indicates the maximum supported number of tile columns minus 1.
+     * When app sets the tile column number equaling to OptMaxTileColumnsMinus1 +1, the performance could be
```

Comment:
@eshiryae , if the tile columns number equaling to (OptMaxTileColumnsMinus1 + 1) . is there also performance benefit?

Reply:
yes, in that case the performance could be optimized due to parallel processing